programs/ssl/mini_client.c

```diff
@@ -166,6 +166,16 @@ int main(void)
     mbedtls_ssl_config conf;
     mbedtls_ctr_drbg_init(&ctr_drbg);
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    psa_status_t status = psa_crypto_init();
+    if (status != PSA_SUCCESS) {
+        mbedtls_fprintf(stderr, "Failed to initialize PSA Crypto implementation: %d\n",
```

Comment:
This program does not use `printf()` in other places, and that's on purpose as it's used for some size measurements, where we don't want to pull in a large part of the standard library. So I think we should skip this line here, just for this program.